### PR TITLE
Fixed: A_LookEx did not account for the +LOOKALLAROUND flag.

### PR DESCRIPTION
**Pull request by madame-rachelle**
(opened on behalf of another person)

### CODE REVIEW

**Comment by inkoalawetrust** [comment]
Why did all the CI checks fail on this simple code change ?

File: src/playsim/p_enemy.cpp [hunk 1]
```diff
@@ -1271,7 +1271,7 @@ int P_IsVisible(AActor *lookee, AActor *other, INTBOOL allaround, FLookExParams
 	{
 		maxdist = params->maxDist;
 		mindist = params->minDist;
-		fov = params->Fov;
+		fov = allaround ? 0 : params->Fov; // [RK] Account for LOOKALLAROUND flag.
```
[the remark this answers is not in the document]
Comment (madame-rachelle) [reply src/playsim/p_enemy.cpp:1274]: @Ru5tK1ng

[the remark this answers is not in the document]
Comment (Ru5tK1ng) [reply src/playsim/p_enemy.cpp]: Made the change as suggested.

**Comment by nashmuhandes** [comment]
@Ru5tK1ng hi, it appears that this PR broke something. Please read this thread: https://forum.zdoom.org/viewtopic.php?t=79033

**Pull request merged**